script.js

```diff
@@ -16,3 +16,320 @@ navItem.forEach((list) => {
+const mainModalContainer = document.querySelector('.modal-window');
+const wholeCont = document.querySelector('.background-blur');
+
+const projects = [
+  {
+    featuredImage: 'images/Snapshoot-Portfolio.png',
+    name: 'Tonic',
+    information: ['Canopy', 'Back End Dev', '2015'],
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['Html', 'css', 'JavaScript'],
+    modalName: 'Tonic',
+    demo: 'https://chepkok3.github.io/Personal-Portfolio/#',
+    source: 'https://github.com/chepkok3/Personal-Portfolio',
+  },
+  {
+    featuredImage: 'images/Snapshoot-Availability.png',
+    name: 'Tonic',
+    information: ['Canopy', 'Back End Dev', '2015'],
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    modalName: 'Tonic',
+    technologies: ['Html', 'css', 'JavaScript'],
+    demo: 'https://chepkok3.github.io/Personal-Portfolio/#',
+    source: 'https://github.com/chepkok3/Personal-Portfolio',
+  },
+  {
+    featuredImage: 'images/Snapshoot-Gymfit.png',
+    name: 'Tonic',
+    information: ['Canopy', 'Back End Dev', '2015'],
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['Html', 'css', 'JavaScript'],
+    modalName: 'Tonic',
+    demo: 'https://chepkok3.github.io/Personal-Portfolio/#',
+    source: 'https://github.com/chepkok3/Personal-Portfolio',
+  },
+  {
+    featuredImage: 'images/Snapshoot-Printing.png',
+    name: 'Tonic',
+    information: ['Canopy', 'Back End Dev', '2015'],
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['Html', 'css', 'JavaScript'],
+    modalName: 'Tonic',
+    demo: 'https://chepkok3.github.io/Personal-Portfolio/#',
+    source: 'https://github.com/chepkok3/Personal-Portfolio',
+  },
+];
+
+// Render mobile portfolio
+projects.forEach((project, index) => {
+  let tech = '';
+  project.technologies.forEach((techno) => {
+    tech += `
+    <li class="btn">
+     ${techno}
+    </li>`;
+  });
+
+  let info = '';
+  project.information.forEach((infoItem) => {
+    info += `<li class="canopy">
+     ${infoItem}
+    </li>`;
+  });
+
+  const eachProject = `
+  <div class="mobile-cards">
+  <div class="project-card">
+  <img
+    class="snapshot-portfolio-img"
+    src=${project.featuredImage}
+    alt="snapshot-portfolio"
+  />
+
+  <div class="project-text">
+    <h2 class="project-title">
+    ${project.name}</h2>
+    <div class="counter">
+    ${info}
+    </div>
+
+    <div class="primary-text">
+    ${project.description}
+    </div>
+    <div class="languages">
+      <ul class="programming-languages">
+      ${tech}
+      </ul>
+    </div>
+    <div class="text-see-project">
+      <a class="see-project-button" id=${index} href="#">See Project</a>
+    </div>
+  </div>
+  </div>
+  </div>
+  <div>
```

Comment:
**OPTIONAL** - Your third project card seems to be causing a huge scroll on your page. Please fix this to help improve the UI of your portfolio. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your Project:
<img width="947" alt="iiihih" src="https://user-images.githubusercontent.com/73607512/210741928-70a5eaef-f754-4e56-902b-dc69e8f13cb1.PNG">

Reply:
@mckent05 Thank you the optional suggestion. I will take it into consideration and make the necessary changes.🙏